modules/programs/zathura.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ...}:
+
+with lib;
+
+let
+
+  cfg = config.programs.zathura;
+
+  formatLine = n: v:
+    let
+      formatValue = v:
+        if isBool v then (if v then "true" else "false")
+        else toString v;
+    in
+      "set ${n}\t\"${formatValue v}\"";
+in
+
+{
+  meta.maintainers = [maintainers.rprospero];
+
+  options.programs.zathura = {
+    enable = mkEnableOption ''Zathura is a highly customizable and funtional
+      document viewer focused on keyboard interaction.'';
+    options = mkOption {
+      default = null;
```

Comment:
Can simply make this `default = {}`.

Reply:
I did not know that null and {} where synonyms.  I've learned more about nix messing with home-manager over the past week than I've learned in the past year of messing with my configuration.nix file.

Reply:
They are not synonyms. It's just that I think `{}` is a more sensible default 😉

In general I think it's best to avoid `null` unless it really is needed.